plugins/Eq/EqControlsDialog.cpp

```diff
@@ -55,10 +55,10 @@ EqControlsDialog::EqControlsDialog( EqControls *controls ) :
 
 	EqSpectrumView * inSpec = new EqSpectrumView( &controls->m_inFftBands, this );
 	inSpec->move( 26, 17 );
-	inSpec->setColor( QColor( 54, 45, 142, 150 ) );
+	inSpec->setColor( QColor( 63, 202, 141, 245 );
```

Comment:
A `(` is missing...

Reply:
oof, lemmie edit

Reply:
fixed, how the heck did i do that 🤔

Reply:
wait a sec
why is it not displaying

Reply:
o wait, its under xD
i need to figure out this pull request thing